lib/hobo_fields/model/field_spec.rb

```diff
@@ -109,8 +109,13 @@ def different_to?(col_spec)
             check_attributes << :limit if sql_type.in?([:string, :text, :binary, :varbinary, :integer, :enum])
             check_attributes.any? do |k|
               if k == :default
-                cast_type = ActiveRecord::Base.connection.lookup_cast_type_from_column(col_spec) or raise "cast_type not found for #{col_spec.inspec}"
-                cast_type.deserialize(col_spec.default) != cast_type.deserialize(default)
+                case Rails::VERSION::MAJOR
+                when 4
+                  col_spec.type_cast_from_database(col_spec.default) != col_spec.type_cast_from_database(default)
+                else
+                  cast_type = ActiveRecord::Base.connection.lookup_cast_type_from_column(col_spec) or raise "cast_type not found for #{col_spec.inspec}"
```

Comment:
Hmm, shouldn't that be `.inspect` with a `t`?!! 🤦

Reply:
It sure should.  More proof that the test coverage here is bad.

Reply:
fixed

Reply:
You know what, I think I just added that `or` part! 🤦 🤦 . Previously the code would just stumble on the `nil` and blow up a few lines later. We're planning to add real test coverage when Donnie and I work on it.